kibana/install.sh

```diff
@@ -6,10 +6,10 @@ set -o pipefail
 
 source "${HOME}/prep-install.${RELEASE_STREAM}"
 
-ORIGIN_KIBANA_PLUGIN=$(ls -t1 ${HOME}/origin-kibana-v$KIBANA_VER-*.zip | head -1)
+ORIGIN_KIBANA_PLUGIN=$(ls -t1 ${HOME}/origin-kibana-v${KIBANA_VER}*.zip | head -1)
 ${KIBANA_HOME}/bin/kibana-plugin install file://${ORIGIN_KIBANA_PLUGIN}
 
 chmod -R og+w "${HOME}"
 chmod -R og+rw "${KIBANA_HOME}"
-chmod -R og+rw /var/lib/kibana
+mkdir -m og+w -p /var/lib/kibana ||
```

Comment:
Why would this fail?  that is, why do you need the `||`?

Reply:
if yo udont have permissions to do so?  but i guess if you are building the image perms are out the window... or if the directory already exists?

Reply:
removed